guides/products-and-variants/variants.md

```diff
@@ -56,37 +56,53 @@ values to each variant.
 
 ## Master variants
 
-Every product has a master variant. By default, the master variant is the first
-variant created for a product. When additional variants are created, they
-inherit the properties of the master variant until their unique properties are
-set.
+Every product has a master variant. By default, the master variant is first
```

Comment:
`the master variant is first` => `the master variant is *the* first`

Reply:
:+1: Thanks. Rebased with this change.